api/logbroker.proto

```diff
@@ -80,6 +86,10 @@ message LogMessage {
 
 	// Data is the raw log message, as generated by the application.
 	bytes data = 4;
+
+	// Attrs is a list of key value pairs representing additional log details
+	// that may have been returned from the logger
+	repeated LogAttr attrs = 5;
```

Comment:
Can you add `[(gogoproto.nullable) = false]` to this line? This gives a slice of `LogAttr`s instead of a slice of pointers to `LogAttr`s